packages/xo-web/src/xo-app/backup-ng/index.js

```diff
@@ -190,13 +190,18 @@ class JobsTable extends React.Component {
       },
       {
         itemRenderer: job => {
-          const { concurrency, offlineSnapshot } = job.settings[''] || {}
+          const { concurrency, offlineSnapshot, reportWhen, timeout } =
+            job.settings[''] || {}
 
           return (
             <Ul>
+              {reportWhen && <Li>{_.keyValue(_('reportWhen'), reportWhen)}</Li>}
               {concurrency > 0 && (
                 <Li>{_.keyValue(_('concurrency'), concurrency)}</Li>
               )}
+              {timeout && (
```

Comment:
Do explicit tests for non boolean values :slightly_smiling_face:

Reply:
I'ts good ?